manila-cli.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
```

Comment:
Instead of using a top-level script like this, create a `scality_manila_utils.cli` module with a `main(args=None)` procedure, and add a `console_script` using `setuptools`. You can add and `if __name__ == '__main__': main()` in there to be able to launch it for testing purposes, then retrieve `sys.argv` if `args` is `None`, or use the provided args for parsing etc, which aids writing unit tests. See https://github.com/scality/ScalitySproxydSwift/commit/49512f4d09dfe0280cc8c9e4e23eec5a8bebf0ba for an example.